Wireframe/css/style.css

```diff
@@ -0,0 +1,71 @@
+*{
+    text-align: center;
+    font-family: 'Courier New', Courier, monospace;
+    
+   
+}
+
+header{
+    background-color: rgba(255, 98, 0, 0.432);
+    color: white;
+    padding: 9px;
+    margin-bottom: 10px;
+}
+
+main{
+    display: grid;
+    gap:10px;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: auto auto;
+}
+
+.one{
+    border: 2px solid #ddd;
+    grid-column: span 2;
+    padding: 20px;
+}
+
+
+
+.two{
+    border: 2px solid #ddd;
+    grid-column: span 1;
+    grid-row: span 1;
+    padding: 20px;
+}
+
+.three{
+    border: 2px solid #ddd;
+    grid-row: span 1;
+    grid-column: span 1;
+    padding: 20px;
+}
```

Comment:
Super slick styles for the articles here! 
You can notice that all three of the classes have similar styles. Moreover, `.two` and `three` have the same styles.

There's one more requirement we need to meet here:

> The first article is styled using the :first-child pseudo-class.

Thus, this is a great opportunity for optimization! What do you think if we create one common style for all articles and use ` :first-child` pseudo class to highlight the first one?